packages/mergebot/src/_tests/fixtures/43136/mutations.json

```diff
@@ -52,14 +52,5 @@
         "body": "🔔 @RReverser — please [review this PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43136/files) in the next few days. Be sure to explicitly select **`Approve`** or **`Request Changes`** in the GitHub UI so I know what's going on.\n<!--typescript_bot_pinging-reviewers-->"
       }
     }
-  },
-  {
-    "mutation": "mutation ($input: AddCommentInput!) {\n  addComment(input: $input) {\n    __typename\n  }\n}\n",
-    "variables": {
-      "input": {
-        "subjectId": "MDExOlB1bGxSZXF1ZXN0Mzg4MzgyMzYy",
-        "body": "@RReverser Thank you for reviewing this PR! The author has pushed new commits since your last review. Could you take another look and submit a fresh review?\n<!--typescript_bot_stale-ping-f5a3ad-37d784d-->"
```

Comment:
This is an example of a downside; if randos do come and review, anything they say will be ignored and they won't be pinged.

In this case, I doubt RReverser is a rando 😄